src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -18,10 +11,17 @@
 import com.google.maps.model.Geometry;
 import com.google.maps.model.PlacesSearchResponse;
 import com.google.maps.model.PlacesSearchResult;
-import java.util.Iterator;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.LinkedList;
+import java.util.Arrays;
+import java.io.Serializable;
+import java.net.URL;
+import java.util.logging.Logger;
+import java.util.Iterator;
```

Comment:
Most of these are not being used. Please review.

Reply:
I fixed this, but I was planning to go through all the import statements in another pull request and remove the ones that were not used. I went through all the files and deleted the import statements that were not used.